apps/desktop/src/components/editor-area/note-header/listen-button.tsx

```diff
@@ -83,7 +80,8 @@ export default function ListenButton({ sessionId }: { sessionId: string }) {
     );
   }
 
-  if (ongoingSessionStatus === "running_paused") {
+  // Show Resume button only if the current note is the one that's paused
+  if (ongoingSessionStatus === "running_paused" && sessionId === ongoingSessionId) {
```

Comment:
if condition already explain "what" it does.
Comment should be only added if adding "Why" is necessary.